File: Backend/analyse_service/src/main/java/com/example/analyse_service/entity/TestAnalyse.java
package com.example.analyse_service.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class TestAnalyse {
    private Long id;
    private Long idAnalyse;          // L'ID de l'analyse associée
    private String nomTest;          // Nom du test
    private String sousEpreuve;      // Sous-épreuve du test
    private Double intervalMinDeReference; // Intervalle min de référence
    private Double intervalMaxDeReference; // Intervalle max de référence
    private String uniteDeReference;   // Unité de référence
    private String details;
}

File: angular-front/src/app/tests/TestAnalyse.ts
export interface TestAnalyse {
    id?: number;                    // L'ID unique du test
    idAnalyse: number;             // L'ID de l'analyse associée
    nomTest: string;               // Nom du test
    sousEpreuve: string;           // Sous-épreuve du test
    intervalMinDeReference: number; // Intervalle min de référence
    intervalMaxDeReference: number; // Intervalle max de référence
    uniteDeReference: string;      // Unité de référence
    details: string;               // Détails supplémentaires
  }
